projects/cashmere/src/lib/radio-button/radio-button.md

```diff
@@ -1,28 +1,32 @@
-##### Radio Text Formatting
+##### Button Toggle Text Formatting
```

Comment:
This is just replacing the radio buttons then? If so, it would probably be good to rename the files to be more consistent

Reply:
nice catch! that got renamed on accident as I didn't change any other components intentionally

Reply:
Haha, OK cool 😆 This file's going to be totally reverted then?

Reply:
Man I don't remember making that change  - but it shouldn't say that regardless lol. It will be fixed as soon as I can get this past this linting error with codelyzer